scopus-import/src/test/java/no/sikt/nva/scopus/conversion/PiaConnectionTest.java

```diff
@@ -64,12 +65,12 @@ void shouldSelectTopLevelOrgWithHighestCountNumber() {
     }
 
     @Test
-    void shouldReturnNullWhenAffiliationListFromPiaIsEmpty() {
+    void shouldReturnOptionalEmptyWhenAffiliationListFromPiaIsEmpty() {
```

Comment:
Should it though?

Reply:
The sad path where PIA doesn't respond is what we expect to happen. Our attitude is that we will be delightfully surprised if PIA cough up something of value.